packages.tsv

```diff
@@ -256,4 +256,4 @@ conda	t_coffee	11.0.8--py35_1	t_coffee --help
 conda	piranha	1.2.1--gsl1.16_0	Piranha
 conda	piranha	1.2.1--gsl1.16_2	Piranha
 conda	trimmomatic	0.36--3	trimmomatic --help
-conda bedops=2.4.19 bedops
+conda bedops  2.4.19  bedops  
```

Comment:
there is still a space :)
All columns needs to be separated by tabs.

Moreover the real version should be `2.4.19--0` the `--0` it the build number.
Let us know how we can make it more obvious.